docs/developers/optimistic-oracle-integration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Integrating with the Optimistic Oracle
+sidebar_label: Integrating with the Optimistic Oracle
+---
+
+## "Deposit Box" Example Contract
+
+This example will set up an `OptimisticDepositBox` contract which custodies a user’s ERC-20 token balance.
+
+On a local testnet blockchain, the user will deposit wETH (Wrapped Ether) into the contract and withdraw wETH corresponding to a desired USD amount.
+
+The user links the `OptimisticDepositBox` with one of the price identifiers enabled on the DVM.
+
+In this example, the user will deposit wETH into the `OptimisticDepositBox` and register it with the `ETH/USD` price identifier.
+
+The user can now withdraw a USD-denominated amount of wETH from their `DepositBox` via smart contract calls. The feature introduced by the Optimistic Oracle is optimistic on-chain pricing of the user's ERC-20 balance.
+
+In this example, the user would not have been able to transfer USD-denominated amounts of wETH without referencing an off-chain `ETH/USD` price feed.
+
+The Optimistic Oracle therefore enables the user to "pull" a reference price.
+
+Unlike price requests to the DVM, a price request to the Optimistic Oracle can be resolved within a specified liveness window if there are no disputes, which can be significantly shorter than the DVM voting period. The price requestor is not currently required to pay fees to the DVM.
+
+The requestor can offer a reward for the proposer who responds to a price request, but the reward value is set to `0` in this tutorial example.
+
+The price proposer posts a bond along with their price, which will be refunded if the price is not disputed, or if a dispute is resolved in the proposer's favor. Otherwise, this bond is used to pay the final fee to the DVM and pay a reward to a successful disputer.
+
+While the example below uses wETH as the currency for deposits to the `OptimisticDepositBox` and ETH/USD as the price identifier, the code supports use of any ERC-20 token as the currency for deposits.
```

Comment:
Like this! But can the content including/following this sentence be a concluding section at the end after the tutorial since it is more about "next steps" rather than this specific tutorial?